pypeit/images/rawimage.py

```diff
@@ -564,6 +567,8 @@ def spatial_flexure_shift(self, slits, force=False):
             return
```

Comment:
If the warning is correct and spatial flexure shift has already been calculated, should this return self.spat_flexure_shift?

Reply:
Nice catch!  Edited.